packages/storybook/examples/core/Button.stories.js

```diff
@@ -0,0 +1,262 @@
+import React from 'react';
+import { action } from '@storybook/addon-actions';
+
+import Button, { PureButton } from '@ichef/gypcrete/src/Button';
+import FlexRow from 'utils/FlexRow';
+
+export default {
+    title: '@ichef/gypcrete|Button',
+    component: PureButton,
+    subcomponents: {
+        'rowComp()': Button,
+    },
+};
+
+export function BasicUsage() {
+    return (
+        <FlexRow>
+            <Button
+                bold
+                basic="Black Button"
+                aside="Default color"
+                tag="Tag"
+                icon="add"
+                onClick={action('clicked')} />
+
+            <Button
+                color="blue"
+                basic="Blue"
+                aside="Variants"
+                tag="Tag"
+                icon="add" />
+
+            <Button
+                color="red"
+                basic="Red"
+                aside="Variants"
+                tag="Tag"
+                icon="add" />
+
+            <Button
+                color="white"
+                basic="White"
+                aside="Variants"
+                tag="Tag"
+                icon="add" />
+        </FlexRow>
+    );
+}
+
+export function ButtonWithStatus() {
+    return (
+        <FlexRow>
+            <Button
+                basic="Loading"
+                tag="Tag"
+                icon="add"
+                status="loading" />
+
+            <Button
+                basic="Success"
+                tag="Tag"
+                icon="add"
+                status="success"
+                statusOptions={{ autohide: false }} />
+
+            <Button
+                basic="Error"
+                tag="Tag"
+                icon="add"
+                status="error"
+                errorMsg="Save failed" />
+        </FlexRow>
+    );
+}
+
+function NormalButton(props) {
+    return (
+        <Button
+            basic="Button"
+            aside="<button>"
+            tag="Tag"
+            tagName="button"
+            {...props} />
+    );
+}
+
+function DivButton(props) {
+    return (
+        <Button
+            basic="Button"
+            aside="<div>"
+            tag="Tag"
+            tagName="div"
+            {...props} />
+    );
+}
+
+function LinkButton(props) {
+    return (
+        <Button
+            basic="Button"
+            aside="<a>"
+            tag="Tag"
+            tagName="a"
+            // props for <a>
+            href="http://ichefpos.com/"
+            target="_blank"
+            {...props} />
+    );
+}
+
+export function CustomTagButton() {
```

Comment:
I suggest to use `<Button>` here instead of using the wrapped variants.
That way when you click “show code” you can actually see how to customize the tag.
<img width="916" alt="截圖 2020-02-05 下午3 45 36" src="https://user-images.githubusercontent.com/365035/73821291-9133ad80-482e-11ea-9feb-1d45dfba9f64.png">

Besides I believe we can reduce the examples to just one `<Button>` per tag.